draft-demarco-oauth-status-attestations.md

```diff
@@ -384,7 +385,8 @@ When the JWT or CWT format are used, the JWT/CWT MUST contain the parameters def
 
 | Payload | Description | Reference |
 | --- | --- | --- |
-| **iss** | Wallet identifier. | {{RFC9126}}, {{RFC7519}} |
+| **iss** | Wallet identifier. The Wallet identifier is out of the scope of this specs considering that it may be just a
```

Comment:
the wallet identifier is not a framework

Reply:
The last sentence ("considering that....") is not clear to me. Is there a reason why we decided to emphasize that the Wallet identifier is out of the scope of this spec? I suggest removing this sentence., if you agree.
@OR13 @marinaado

Reply:
@fmarino-ipzs the way a wallet can be identified or the value to be supposed to be used for identify a wallet is definitively out of the scope of this spec